README.md

```diff
@@ -1,2 +1,55 @@
 # prj-jse-bookstore
-Desktop application to automate a book store
+Desktop application to automate a book store.
+
+## How it works?
+It's very simple to use, these are some instructions:
+
+**_Menu:_** 
+   - At the menu you can access the options for:
+
+**1. Products:**
```

Comment:
You have a nice feature to control stock, you should mention about this somewhere